aict_tools/scripts/plot_disp_performance.py

```diff
@@ -0,0 +1,126 @@
+import click
+import logging
+import matplotlib
+import matplotlib.pyplot as plt
+import joblib
+import fact.io
+
+from ..configuration import AICTConfig
+from ..plotting import (
+    plot_roc,
+    plot_probabilities,
+    plot_regressor_confusion,
+    plot_bias_resolution,
+    plot_feature_importances,
+    plot_true_delta_delta
+)
+
+if matplotlib.get_backend() == 'pgf':
+    from matplotlib.backends.backend_pgf import PdfPages
+else:
+    from matplotlib.backends.backend_pdf import PdfPages
+
+
+@click.command()
+@click.argument('configuration_path', type=click.Path(exists=True, dir_okay=False))
+@click.argument('performance_path', type=click.Path(exists=True, dir_okay=False))
+@click.argument('data_path', type=click.Path(exists=True, dir_okay=False))
+@click.argument('sign_model_path', type=click.Path(exists=True, dir_okay=False))
```

Comment:
The order here is different than in the train/apply scripts, we should be consistent (first disp then sign)